Xext/security.c

```diff
@@ -430,6 +435,15 @@ ProcSecurityGenerateAuthorization(ClientPtr client)
         return BadValue;
     }
 
+    if (client->swapped) {
+        swaps(&stuff->nbytesAuthProto);
+        swaps(&stuff->nbytesAuthData);
+        swapl(&stuff->valueMask);
+
+        unsigned long nvalues = (((CARD32 *) stuff) + client->req_len) - values;
```

Comment:
I guess this can be simplified (using the expression for `values`).

Reply:
`values` needs to be computed anyways. So I'd prefer leaving it for later refactoring of the request parsing.
this PR shall only inline the SProc*s for now